apps/prairielearn/assets/scripts/question.ts

```diff
@@ -229,6 +292,33 @@ function setupDynamicObjects() {
   }
 }
 
+function loadPendingSubmissionPanel(this: HTMLDivElement) {
+  const { submissionId, dynamicRenderUrl } = this.dataset;
+  if (submissionId == null || dynamicRenderUrl == null) return;
+
+  const wasModalOpen = ($(`#submissionInfoModal-${submissionId}`).data('bs.modal') || {})._isShown;
+  $(`#submissionInfoModal-${submissionId}`).modal('hide');
```

Comment:
I know this is just coming from the existing code, but I can't understand how this is necessary. How would it be possible for someone to expand a past submission _while_ the modal is visible? Wouldn't the modal prevent them from interacting with the submissions at all?

Reply:
I believe this was copied from the behaviour in the external grader socket, where this is a possibility. Indeed I noticed that this was not possible to replicate during testing, I can remove it.

Reply:
Removed in 7d1ff914f9d8a42f893a698427e9faa31c44b9bd.